proms-resources/clinical-data/pom.xml

```
@@ -42,6 +42,7 @@
             <Sling-Nodetypes>SLING-INF/nodetypes/proms.cnd</Sling-Nodetypes>
             <Include-Resource>{maven-resources},src/main/media</Include-Resource>
             <Sling-Initial-Content>
+              SLING-INF/content/Extensions/;path:=/;overwrite:=true,
```

```suggestion
              SLING-INF/content/Extensions/DashboardViews/;path:=/Extensions/DashboardViews/;overwrite:=true,
              SLING-INF/content/Extensions/Sidebar/;path:=/Extensions/Sidebar/;overwrite:=true
```

Also these should be moved to the bottom for consistency with other InitialContent declarations.